src/components/orders/DetailsTable/index.tsx

```diff
@@ -170,11 +211,18 @@ export function DetailsTable(props: Props): JSX.Element | null {
                 {areTradesLoading ? (
                   <Spinner />
                 ) : txHash ? (
-                  <RowWithCopyButton
-                    textToCopy={txHash}
-                    onCopy={(): void => onCopy('settlementTx')}
-                    contentsToDisplay={<LinkWithPrefixNetwork to={`/tx/${txHash}`}>{txHash}</LinkWithPrefixNetwork>}
-                  />
+                  <Wrapper>
+                    <RowWithCopyButton
+                      textToCopy={txHash}
+                      onCopy={(): void => onCopy('settlementTx')}
+                      contentsToDisplay={
+                        <LinkWithPrefixNetwork to={`/tx/${txHash}/?tab=graph`}>{txHash}</LinkWithPrefixNetwork>
```

Comment:
Sorry if I wasn't clear, this link shouldn't be changed, I was referring to the `View batch graph` button that needs to be a link that points to `/tx/${txHash}/?tab=graph`